db/migrate/20200411210313_create_peeps.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePeeps < ActiveRecord::Migration[6.0]
+  def change
+    create_table :peeps do |t|
+      t.string :content
+      t.timestamps
+    end
+  end
+end
```

Comment:
Readability: could there be a clearer name for t - although the scope is short so maybe t is perfectly fine - what do you think ?